### DISCUSSION TITLE
Homepage Room ID generates "invalid url" error.

### DISCUSSION THREAD

**Discussion by NovySan** [mozilla/hubs#3873] [Hubs Cloud Questions] [answered]
After setting the Homepage Room ID in the App Settings > Features > Homepage Room ID, visiting the homepage now shows only "invalid url." Does this take a few minutes to become active or something?

**Comment by NovySan** [comment]
Should I move this over to Issues?

**Comment by robinkwilson** [comment, marked as answer]
Double check that the Homepage Room ID is the ID of the room, _**not**_ the full URL. 

This worked for me in my instance:
![image](https://user-images.githubusercontent.com/29560735/107715187-928a8800-6c83-11eb-9a82-bf89c8ba562f.png)

**Reply by robinkwilson** [reply]
Marking as answered for now, if this doesn't fix your issue, please unmark!
